Sources/Backup/Commands/OADeleteAllFilesCommand.mm

```diff
@@ -107,9 +107,15 @@ - (void)doInBackground
             {
                 [filesToDelete addObjectsFromArray:remoteFiles];
             }
-            [self publishProgress:filesToDelete];
```

Comment:
No point in this change, if files to delete is empty you still pass an empty array